### DISCUSSION TITLE
How do I express project open data (data.json) as a dataset?

### DISCUSSION THREAD

**Discussion by dafeder** [frictionlessdata/frictionlessdata.io#585] [Q&A]
I'm working on a data catalog that primarily stores and displays metadata using the [Project Open Data](https://project-open-data.cio.gov/v1.1/schema/) metadata schema. What is the easiest way for me to present these datasets as data packages that will integrate with other frictionless data tools?

For instance, here is a one-dataset catalog example: https://gist.github.com/philipashlock/21ff607527863fba200b

**Comment by rufuspollock** [comment]
@dafeder great question and this fits perfectly with current mapping efforts in frictionlessdata/forum#53 

The starting point would be to extract the json schema fields and put in the spreadsheet here: https://docs.google.com/spreadsheets/d/1XdqGTFni5Jfs8AMbcbfsP7m11h9mOHS0eDtUZtqGVSg/edit#gid=774748003

Great if this is something you can help with o/w my colleagues @sglavoie may be able to help with it.

Once that is done we can do a mapping which is generally pretty easy ~10m or so. Basically, most stuff just transfers across.

**Comment by sglavoie** [comment]
As a quick update, we have now described the Project Open Data schema in the spreadsheet ([in this sheet](https://docs.google.com/spreadsheets/d/1XdqGTFni5Jfs8AMbcbfsP7m11h9mOHS0eDtUZtqGVSg/edit#gid=343209482)). :tada:

**Comment by dafeder** [comment]
Thanks @rufuspollock and @sglavoie, coming back to this after a bit of time way. I'm going to go through but it looks pretty complete. One thing that's nice -- the move to calling this standard "DCAT-US" instead of the clunky "Project Open Data Schema" seems to be official, so that can make these docs a little more manageable. I'll try to create a mapping table as an additional sheet.

**Comment by rufuspollock** [comment]
@dafeder that's great 👏  - let us know when you have gone through the mapping setup.

**Comment by aborruso** [comment]
Hi all. Currently is there some way to convert (sort of) from DCAT to frictionless and viceversa?

Thank you

**Reply by augusto-herrmann** [reply]
Not yet, but there is [a mapping effort](https://github.com/frictionlessdata/project/issues/551) underway to associate metadata in data packages to/from DCAT2. If you'd like to, you can help us finish or revise the mapping once it's done.
